.github/actions/setup-torus/action.yml

```diff
@@ -0,0 +1,36 @@
+name: "Create Torus properties file"
+description: "Create torus.properties and fill them"
+inputs:
+  torus-verifier-debug:
```

Comment:
we need debug because we have ktlint which formats all build types and unit tests. 
I could set it as empty but the code is becoming complex.
I decided that it's better just to add the debug one